app/controllers/browse.js

```diff
@@ -3,6 +3,7 @@ import config from "../config/environment";
 
 export default Ember.Controller.extend({
   isMobileApp: config.cordova.enabled,
+  packageCategory: Ember.inject.controller(),
```

Comment:
How come we need the packageCategory controller ?
I feel like injecting a controller into another is generally a code smell, but I also know that we're sometimes forced to (been guilty of that myself).
So just want to know the reason, maybe we can come up with something better 😄

Reply:
I want to set a property of packageCategory from this controller.

Reply:
maybe a service can be created for this, and this code can be moved to the service to avoid controller injection. Do let me know your thoughts on this @patrixr .